### DISCUSSION TITLE
Error loading key "/tmp/git-resource-private-key": invalid format

### DISCUSSION THREAD

**Discussion by ln-rinterface** [concourse/concourse#8238] [General]
Not sure why I am getting this error as I have done this a bunch of times just fine. Installed new instance of this in a dev env in AWS. I added the secret to secrets mgmr and I get 
Error loading key "/tmp/git-resource-private-key": invalid format

I tested the key directly in my yml file and it works just fine, but when adding it to aws secrets it says the formats bad? its the same copy paste. Im going to check if its bc the worker cannot access secrets but that has not changed. 

Concourse: 7.7.1
containerd runners
no DNS issue here either - that was checked as well

Confirmed secret is allowed to be pulled from instance

**Comment by ZhudongVm** [comment]
My advice is to check your yml file again, Do not forget add "|"  after "private_key: ";
